.tidyallrc

```diff
@@ -2,5 +2,5 @@
 ; Notice that this does include also non-Perl files in script/ so it is
 ; suggested to use tools/tidy to cover all appropriate files. "tidyall -a"
 ; will try to cover too much and fail on the non-Perl files
-select = **/*.{pl,pm,t} script/*
+select = {lib,t}/**/*.{pl,pm,t} script/*
```

Comment:
This would exclude tools/ at least. I suggest to use

```suggestion
ignore = external/
```

see https://metacpan.org/pod/tidyall

Reply:
Currently `./tools/tidy -l` does not list anything in tools, but ok.

But I'm a fan of explicitly including things. I hate that I have to use `ignore` in os-autoinst. So I will include `tools` explicitly.

Reply:
Well, I prefer to include everything and only exclude so that any new files or directories are not silently ignored by mistake. Having too many files included produces quite explicit errors.

Reply:
So one against one. Any other opinions?

Reply:
How many new directories in the root directory containing perl files are we expecting in the future?

Reply:
That's the point, I don't know. To be safe I suggest to include all and specify the exceptions

Reply:
There are some standard directories where perl files live in a project, and then maybe some custom ones. This config file can serve as an easy way to document in which directories we expect perl files that we want to tidy.
I don't get the argument of "being safe", especially when talking about perltidy.

Reply:
Just convince someone else of the team to vote for the "ignore" option and I'll do it =)